docs/about.md

```diff
@@ -61,7 +61,9 @@ What's more, when the creator is free to use their judgment, many new kinds of p
 
 ### **What kind of betting system do you use?**
 
-Manifold Markets uses a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
+Manifold markets is currently using more than one kind of automated market maker.
+
+Markets created before March 15 and Free response markets use a special type of automated market marker based on a dynamic pari-mutuel (DPM) betting system.
```

Comment:
I'd reword this:

"Free response markets, as well as legacy binary markets (created before Mar 15, 2022), use a dynamic..."

Reasons:
- Focus on the most important thing first (FR markets use DPM); less awkward wording
- Cut "special"; it's just one of two different kinds.

Reply:
All legacy binary markets were converted to Maniswap. Only FR and numeric markets use DPM.